LearningMissionSimulation/PlayGroundGavril.cs

```diff
@@ -1,14 +1,58 @@
-﻿using System;
+﻿using LearningMissionLab;
+using System;
 using System.Collections.Generic;
 using System.Text;
 
 namespace LearningMissionSimulation
 {
-    public  class PlayGroundGavril
+    public class PlayGroundGavril
     {
-        public PlayGroundGavril()
+        List<Account> pendingAccountList = new List<Account>();
+        List<Student> studentList = new List<Student>();
+        List<Instructor> instructorList = new List<Instructor>();
+        Dictionary<Guid, Account> accountDictionary = new Dictionary<Guid, Account>();
+        public void CreateAccounts(int accountCount)
         {
+            for (int i = 0; i < accountCount; i++)
+            {
+                Account account = ObjectGenerator.GenerateAccount();
 
+                accountDictionary.Add(account.Id, account);
+                if (account.Role == Role.Student)
+                {
+                    Student student = ObjectGenerator.GenerateStudent(account.Id);
+                    studentList.Add(student);
+
+                    if (student.DateOfBirth == new DateTime(8, 9, 1981))
```

Comment:
1. You have to check also the account status before adding it to pendingAccountList.
2. This is wrong: if (student.DateOfBirth == new DateTime(8, 9, 1981)).
--- you have to check only the month.

Reply:
Line 26 is the only line that is different from Garush's implementation. That's OK if you understand and can explain what Garush's code is doing. But I need to see your implementation for line 26. And you have to do it yourself. Please don't ask anyone to help you.